crates/bevy_app/src/app.rs

```diff
@@ -341,9 +591,9 @@ impl App {
     /// Inserts a resource to the current [App] and overwrites any resource previously added of the same type.
     ///
     /// A resource in Bevy represents globally unique data. Resources must be added to Bevy Apps
-    /// before using them. This happens with [`App::insert_resource`].
+    /// before using them. This happens with [`insert_resource`](Self::insert_resource).
     ///
-    /// See also `init_resource` for resources that implement `Default` or [`FromWorld`].
+    /// See also `init_resource` for resources that implement `Default` or `FromResources`.
```

Comment:
This should be `FromWorld` (and should link to the trait like it did previously)